### ISSUE TITLE
Where to get `2021-05-17_dna_r9.4.1_minion_96_29d8704b` model

### ISSUE THREAD

**Issue by Rohit-Satyam** [nanoporetech/dorado#1104]
Hi

I have a FASTQ file generated with MinIon Minknow and the fastq headers contains following model information `2021-05-17_dna_r9.4.1_minion_96_29d8704b`. I am using Clair3 for basecalling and I have to provide the model as input. Where can I get this model file?

Dorado gives me the following error:
```
 ./bin/dorado download --model 2021-05-17_dna_r9.4.1_minion_96_29d8704b
[2024-10-27 21:55:57.546] [info] Running: "download" "--model" "2021-05-17_dna_r9.4.1_minion_96_29d8704b"
[2024-10-27 21:55:57.546] [error] '2021-05-17_dna_r9.4.1_minion_96_29d8704b' is not a valid model
```

**Comment by vellamike** [comment]
Hi, can you clarify how you basecalled this data?

**Comment by malton-ont** [comment]
@Rohit-Satyam,

This was an internal model id from our validation database. It corresponds to the standard `dna_r9.4.1_450bps_fast.cfg` model that shipped with guppy prior to v7.2.
